examples/selfie/README.md

```diff
@@ -0,0 +1,54 @@
+## Self Image
```

Comment:
Hey @fredcido, we have introduced a new template for the readme.md for all our public demo examples based on the feedback we received from developers.

You can see one [here](https://github.com/miroapp/app-examples/tree/main/examples/drag-and-drop) for example.

You can find the template.md [here](https://github.com/miroapp/app-examples/blob/main/README.template.md)

Cc: @horeaporutiu

Reply:
Thanks @anthonyroux , I know the new template, I am preparing the PR for review, it's still in draft.